tensorflow/python/ops/math_ops.py

```diff
@@ -1389,6 +1389,7 @@ def reduce_logsumexp(input_tensor, reduction_indices=None, keep_dims=False,
         reduction_indices,
         keep_dims=True)) + my_max
     if not keep_dims:
+      reduction_indices = [reduction_indices] if isinstance(reduction_indices, int) else reduction_indices
```

Comment:
Change this to:

```
reduction_indices = _ReductionDims(input_tensor, reduction_indices)
```

That's what we do for the other reduction functions in this file.